<?xml version="1.0" encoding="UTF-8" standalone="no"?><testsuites failures="517" name="RESTAPIAutomationSuites" tests="517" time="592.419"><testsuite errors="0" failures="517" name="com.tibco.rest.test.DataDrivenTestCases" skipped="0" tests="517" time="592.419">
  <properties/>
      <testcase name="PROXYERROR-09" time="0.333">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPE-02" time="0.002">
            <failure message="Illegal character in path at index 47: http://10.102.168.226:30001/cms/v1/domainTypes/[]" type="java.net.URISyntaxException">
<![CDATA[java.net.URISyntaxException: Illegal character in path at index 47: http://10.102.168.226:30001/cms/v1/domainTypes/[]
  java.net.URI$Parser.fail(Unknown Source)
  java.net.URI$Parser.checkChars(Unknown Source)
  java.net.URI$Parser.parseHierarchical(Unknown Source)
  java.net.URI$Parser.parse(Unknown Source)
  java.net.URI.<init>(Unknown Source)
  io.restassured.internal.http.URIBuilder.convertToURI(URIBuilder.java:91)
  io.restassured.internal.http.URIBuilder$convertToURI.call(Unknown Source)
  io.restassured.internal.RequestSpecificationImpl.generateRequestUriForLogging(RequestSpecificationImpl.groovy:1193)
  sun.reflect.GeneratedMethodAccessor33.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:174)
  io.restassured.internal.RequestSpecificationImpl.newFilterContext(RequestSpecificationImpl.groovy:1154)
  sun.reflect.GeneratedMethodAccessor32.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1729)
  sun.reflect.GeneratedMethodAccessor65.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1737)
  sun.reflect.GeneratedMethodAccessor64.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.get(RequestSpecificationImpl.groovy:170)
  io.restassured.internal.RequestSpecificationImpl.get(RequestSpecificationImpl.groovy)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:210)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PROXYERROR-05" time="1.629">
            <failure message="Expected value should not be &quot;&quot; : Actual value is &quot;&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should not be "" : Actual value is ""
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertNotEquals(Assertions.java:196)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:520)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PROTOCOLBINDINGSX12-06" time="0.361">
            <failure message="Expected value should be &quot;TUESDAY,FRIDAY&quot; : Actual value is &quot;FRIDAY,TUESDAY&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should be "TUESDAY,FRIDAY" : Actual value is "FRIDAY,TUESDAY"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertEquals(Assertions.java:183)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:508)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="DOMAINTYPE-06" time="0.003">
            <failure message="Illegal character in path at index 47: http://10.102.168.226:30001/cms/v1/domainTypes/[]" type="java.net.URISyntaxException">
<![CDATA[java.net.URISyntaxException: Illegal character in path at index 47: http://10.102.168.226:30001/cms/v1/domainTypes/[]
  java.net.URI$Parser.fail(Unknown Source)
  java.net.URI$Parser.checkChars(Unknown Source)
  java.net.URI$Parser.parseHierarchical(Unknown Source)
  java.net.URI$Parser.parse(Unknown Source)
  java.net.URI.<init>(Unknown Source)
  io.restassured.internal.http.URIBuilder.convertToURI(URIBuilder.java:91)
  io.restassured.internal.http.URIBuilder$convertToURI.call(Unknown Source)
  io.restassured.internal.RequestSpecificationImpl.generateRequestUriForLogging(RequestSpecificationImpl.groovy:1193)
  sun.reflect.GeneratedMethodAccessor33.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:174)
  io.restassured.internal.RequestSpecificationImpl.newFilterContext(RequestSpecificationImpl.groovy:1154)
  sun.reflect.GeneratedMethodAccessor32.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1729)
  sun.reflect.GeneratedMethodAccessor65.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1737)
  sun.reflect.GeneratedMethodAccessor64.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.get(RequestSpecificationImpl.groovy:170)
  io.restassured.internal.RequestSpecificationImpl.get(RequestSpecificationImpl.groovy)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:210)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PCCEC-10" time="0.300">
            <failure message="Expected value should contains &quot;sccode1 you have entered  is already in use&quot; : Actual value is &quot;sccode1 you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "sccode1 you have entered  is already in use" : Actual value is "sccode1 you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:377)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="LOCATIONERROR-04" time="0.394">
            <failure message="Expected value should contains &quot;Admin Office  you have entered  is already in use&quot; : Actual value is &quot;Admin Office  you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "Admin Office  you have entered  is already in use" : Actual value is "Admin Office  you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="DOMAINTYPE-07" time="0.316">
            <failure message="Expected value should contains &quot;Auto Domain Type  you have entered  is already in use&quot; : Actual value is &quot;Auto Domain Type  you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "Auto Domain Type  you have entered  is already in use" : Actual value is "Auto Domain Type  you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PCCEC-06" time="1.698">
            <failure message="Expected value should not be &quot;[]&quot; : Actual value is &quot;[]&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should not be "[]" : Actual value is "[]"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertNotEquals(Assertions.java:196)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:520)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:344)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PROXYERROR-04" time="0.504">
            <failure message="Expected value should be &quot;BCCE-3303&quot; : Actual value is &quot;BCCE-3100&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should be "BCCE-3303" : Actual value is "BCCE-3100"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertEquals(Assertions.java:183)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:508)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="EmailPoller-1" time="1.501">
            <failure message="Expected value should not be &quot;[]&quot; : Actual value is &quot;[]&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should not be "[]" : Actual value is "[]"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertNotEquals(Assertions.java:196)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:520)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:344)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PROXYERROR-07" time="1.729">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXYERROR-02" time="1.248">
            <failure message="Expected value should contains &quot;ProxyHTTP you have entered  is already in use&quot; : Actual value is &quot;ProxyHTTP you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "ProxyHTTP you have entered  is already in use" : Actual value is "ProxyHTTP you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="UserGroup-02" time="0.464">
            <failure message="Expected value should be &quot;Groupname Auto User group  you have entered  is already in use.&quot; : Actual value is &quot;Groupname Auto User group  you have entered is already in use.&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should be "Groupname Auto User group  you have entered  is already in use." : Actual value is "Groupname Auto User group  you have entered is already in use."
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertEquals(Assertions.java:183)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:508)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="ROLE-05" time="0.388">
            <failure message="Expected value should contains &quot;Auto Role Edited  you have entered  is already in use&quot; : Actual value is &quot;Auto Role Edited  you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "Auto Role Edited  you have entered  is already in use" : Actual value is "Auto Role Edited  you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PCCEC-07" time="1.203">
            <failure message="Expected value should contains &quot;Name  you have entered  is already in use&quot; : Actual value is &quot;Name  you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "Name  you have entered  is already in use" : Actual value is "Name  you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:377)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="INBOUNDTRANSPORT-03" time="0.491">
            <failure message="Unable to store response string to variable or couldn't received response." type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Unable to store response string to variable or couldn't received response.
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:415)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="TOutbound8" time="1.327">
            <failure message="Expected value should contains &quot;The Transport name http_rn  you have entered  is already in use&quot; : Actual value is &quot;The Transport name http_rn  you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "The Transport name http_rn  you have entered  is already in use" : Actual value is "The Transport name http_rn  you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:377)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="RNROLEERROR-05" time="1.203">
            <failure message="Expected value should contains &quot;RosettaNet Role Name  you have entered  is already in use&quot; : Actual value is &quot;RosettaNet Role Name  you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "RosettaNet Role Name  you have entered  is already in use" : Actual value is "RosettaNet Role Name  you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PROXYERROR-01" time="1.459">
            <failure message="Unable to store response string to variable or couldn't received response." type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Unable to store response string to variable or couldn't received response.
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:415)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="DOMAINPAIRS-09" time="1.317">
            <failure message="Expected value should be &quot;Domain Identity cannot have empty value.&quot; : Actual value is &quot;Domain Identity cannot have empty value&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should be "Domain Identity cannot have empty value." : Actual value is "Domain Identity cannot have empty value"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertEquals(Assertions.java:183)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:508)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="RNROLEERROR-02" time="1.099">
            <failure message="Expected value should contains &quot;RN Role Test you have entered  is already in use&quot; : Actual value is &quot;RN Role Test you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "RN Role Test you have entered  is already in use" : Actual value is "RN Role Test you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="TOutbound13" time="1.267">
            <failure message="Expected value should contains &quot;The Transport name https_rn  you have entered  is already in use&quot; : Actual value is &quot;The Transport name https_rn  you have entered  is already in use&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "The Transport name https_rn  you have entered  is already in use" : Actual value is "The Transport name https_rn  you have entered  is already in use"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:377)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PROXYERROR-06" time="0.419">
            <failure message="Expected value should contains &quot;Proxy Servers  are set as default&quot; : Actual value is &quot;Proxy Servers  are set as default&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should contains "Proxy Servers  are set as default" : Actual value is "Proxy Servers  are set as default"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertContains(Assertions.java:212)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:533)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:352)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="IBTProtocol-12" time="0.385">
            <failure message="Expected value should be &quot;EMAIL&quot; : Actual value is &quot;[&quot;EMAIL&quot;,&quot;FILE&quot;,&quot;FTP&quot;,&quot;HTTP&quot;]&quot;" type="java.lang.AssertionError">
<![CDATA[java.lang.AssertionError: Expected value should be "EMAIL" : Actual value is "["EMAIL","FILE","FTP","HTTP"]"
  com.tibco.rest.restAssured.Assertions.assertTrue(Assertions.java:166)
  com.tibco.rest.restAssured.Assertions.assertEquals(Assertions.java:183)
  com.tibco.rest.test.DataDrivenTestCases.verifyResponse(DataDrivenTestCases.java:508)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:377)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="DOMAINTYPE-04" time="0.002">
            <failure message="Illegal character in path at index 47: http://10.102.168.226:30001/cms/v1/domainTypes/[]" type="java.net.URISyntaxException">
<![CDATA[java.net.URISyntaxException: Illegal character in path at index 47: http://10.102.168.226:30001/cms/v1/domainTypes/[]
  java.net.URI$Parser.fail(Unknown Source)
  java.net.URI$Parser.checkChars(Unknown Source)
  java.net.URI$Parser.parseHierarchical(Unknown Source)
  java.net.URI$Parser.parse(Unknown Source)
  java.net.URI.<init>(Unknown Source)
  io.restassured.internal.http.URIBuilder.convertToURI(URIBuilder.java:91)
  io.restassured.internal.http.URIBuilder$convertToURI.call(Unknown Source)
  io.restassured.internal.RequestSpecificationImpl.generateRequestUriForLogging(RequestSpecificationImpl.groovy:1193)
  sun.reflect.GeneratedMethodAccessor33.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:174)
  io.restassured.internal.RequestSpecificationImpl.newFilterContext(RequestSpecificationImpl.groovy:1154)
  sun.reflect.GeneratedMethodAccessor32.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1729)
  sun.reflect.GeneratedMethodAccessor65.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1737)
  sun.reflect.GeneratedMethodAccessor64.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
  groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1218)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1027)
  groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:815)
  io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
  org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
  io.restassured.internal.RequestSpecificationImpl.put(RequestSpecificationImpl.groovy:178)
  io.restassured.internal.RequestSpecificationImpl.put(RequestSpecificationImpl.groovy)
  com.tibco.rest.test.DataDrivenTestCases.testcase(DataDrivenTestCases.java:214)
  sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.TestNG.privateMain(TestNG.java:1442)
  org.testng.TestNG.main(TestNG.java:1411)
]]>
      </failure>

          </testcase>
      <testcase name="PARTNER-03" time="1.288">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERX12-02" time="0.619">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-04" time="0.406">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="INSTALLED PROTOCOLS-02" time="1.166">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPE-09" time="1.097">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-10" time="0.409">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-11" time="0.394">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN007" time="0.436">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-08" time="0.391">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-17" time="0.429">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RolesBinding-Search-12" time="0.328">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-OperationsNotifytransaction" time="3.134">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-26" time="0.413">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-OverrideOperationsBinding1" time="0.736">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN004" time="0.310">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-Group01" time="0.675">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="INBOUNDTRANSPORT-04" time="0.479">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-11" time="0.410">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-05" time="1.814">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-03" time="0.432">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-15" time="0.388">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-12" time="0.478">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-03" time="0.345">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-Search-5" time="0.386">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="EnableRNProtocol-9" time="3.277">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RNROLEERROR-07" time="0.432">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-07" time="0.413">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-01" time="2.227">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RNROLEERROR-03" time="0.320">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-05" time="0.359">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-18" time="1.497">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTBA-01" time="1.741">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-02" time="2.498">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SCC-07" time="0.497">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-03" time="1.250">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPE-05" time="0.367">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-07" time="1.611">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-32" time="0.443">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN015" time="1.590">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RBERROR-03" time="1.521">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCode2" time="0.341">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RolesBinding-Search-11" time="0.329">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERLOCATION-02" time="0.584">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-30" time="3.399">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-02" time="3.041">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-20" time="0.491">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RNROLEERROR-04" time="0.412">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-08" time="0.646">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-OverrideOperationsBinding-ost1" time="0.573">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-14" time="0.344">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-01" time="9.260">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="InstalledTProtocols-1" time="1.080">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-02" time="2.416">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOST-04" time="0.383">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-Operation-16" time="0.492">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Host-4" time="1.013">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SCC-06" time="0.342">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Delete Host" time="0.945">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSX12-04" time="1.435">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSX12-05" time="1.397">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-21" time="1.596">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-05" time="0.405">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-20" time="2.640">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSRN-05" time="1.417">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-13" time="1.778">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-02" time="1.463">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-10" time="2.497">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SCC-02" time="0.346">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-21" time="0.562">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateBizAgre-7" time="1.502">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIOn-12" time="0.397">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-18" time="0.523">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RBERROR-05" time="0.355">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTBA-05" time="0.520">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound12" time="2.493">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBD-08" time="2.692">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBDPARTNER-01" time="0.525">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-CreateOperationBinding" time="1.419">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-05" time="0.429">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-Group01" time="1.269">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-14" time="0.441">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-08" time="0.498">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-08" time="0.396">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound15" time="0.498">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-Get-2" time="0.365">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-EnableProtocol-6" time="2.334">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-13" time="0.635">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-04" time="0.462">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-Operation-17" time="0.411">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN009" time="0.910">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-17" time="0.365">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SCC-03" time="1.281">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-07" time="0.569">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="MsgBus-5" time="1.260">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BAErrorCode-05" time="0.597">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-05" time="1.297">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-3" time="2.066">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Create-operation-1" time="1.290">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSX12-01" time="1.816">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-30" time="0.370">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-04" time="2.458">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPE-08" time="0.350">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound10" time="0.340">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-Operation-12" time="0.419">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-07" time="0.569">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound9" time="0.522">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-12" time="0.382">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-07" time="2.830">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-10" time="0.431">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-07" time="0.375">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BAErrorCode-02" time="1.683">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-10" time="0.463">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Host" time="1.750">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN026" time="0.431">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-Group02" time="1.246">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-03" time="0.643">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN017" time="3.517">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-04" time="0.460">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-04" time="1.428">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-01" time="0.507">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-12" time="0.435">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Create-Syncoperation-3" time="2.082">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-09" time="0.370">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BAErrorCode-04" time="1.111">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-02" time="9.482">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TPartner2" time="1.236">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBDHOST-02" time="1.659">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-15" time="0.840">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN001" time="1.459">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GSInfor-3" time="0.446">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-16" time="0.403">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-13" time="0.416">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-11" time="0.371">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Host-4" time="1.184">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSRN-06" time="0.395">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-01" time="1.569">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-06" time="0.433">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="MsgBus-1" time="1.598">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXYERROR-07" time="1.729">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-04" time="0.425">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-08" time="0.623">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATIONERROR-01" time="2.457">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-04" time="0.572">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-05" time="0.493">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GatewayService-2" time="0.445">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="THOST23" time="0.601">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-01" time="1.273">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-04" time="1.631">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBD-04" time="2.405">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RBERROR-04" time="1.376">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-23" time="1.491">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXYERROR-03" time="1.651">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENTX12-01" time="1.485">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound16" time="0.368">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-29" time="0.343">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-12" time="1.213">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BAErrorCode-06" time="1.050">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateBizAgre-7" time="1.460">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-20" time="1.483">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-05" time="0.366">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-01" time="1.723">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Outboundtrans-8" time="2.601">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCode3" time="1.187">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCode1" time="1.145">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TBizAgreement5" time="1.400">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERBA-04" time="0.602">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-18" time="0.387">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-04" time="0.561">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-06" time="0.373">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-11" time="0.899">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-01" time="2.478">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="INBOUNDTRANSPORT-01" time="0.431">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-22" time="0.317">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-17" time="0.349">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-08" time="0.383">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-07" time="0.333">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-16" time="1.950">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN021" time="0.437">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCode5" time="0.390">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-06" time="0.397">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Delete BA" time="0.623">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-02" time="0.419">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SCC-01" time="1.146">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOST-03" time="1.647">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-16" time="0.482">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-ProtocolBinding" time="2.779">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-09" time="0.482">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="INBOUNDTRANSPORT-02" time="0.420">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-05" time="0.506">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-02" time="1.269">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGSX12-02" time="3.739">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOST-02" time="0.399">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-04" time="1.864">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSRN-04" time="0.447">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN002" time="0.558">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-27" time="0.351">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-21" time="2.612">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-03" time="2.217">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN003" time="0.516">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-OverrideOperationsBinding-tt2" time="0.731">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOST-05" time="0.769">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSRN-01" time="0.558">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound17" time="0.382">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GatewayService-1" time="1.541">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-10" time="0.385">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCode6" time="0.333">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-PARTNER-5" time="1.417">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-19" time="0.624">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSRN-02" time="1.104">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-PARTNER-5" time="1.445">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BUSINESSAGREEMENTX12-02" time="0.710">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-2" time="1.745">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-01" time="2.392">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-25" time="0.380">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGSX12-03" time="2.673">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-16" time="0.378">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN022" time="0.364">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-06" time="0.428">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Enableprotocol" time="2.466">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-03" time="0.334">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TPartner22" time="0.741">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-13" time="1.620">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-Get-4" time="0.380">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-19" time="2.543">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-12" time="0.617">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-9" time="1.772">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-09" time="3.248">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-15" time="0.411">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-03" time="10.074">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-28" time="0.429">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-23" time="0.367">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-25" time="0.406">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-09" time="0.385">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-07" time="0.788">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-03" time="1.847">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-10" time="1.572">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBDPARTNER-05" time="0.535">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RNROLEERROR-06" time="0.534">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-6" time="1.635">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-21" time="0.432">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-14" time="0.680">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN011" time="0.417">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-02" time="0.361">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-01" time="2.140">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-03" time="1.907">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-Group02" time="0.618">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-partner-20" time="0.601">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GSInfor-1" time="0.721">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-18" time="1.978">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-1" time="1.806">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-05" time="0.848">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBD-06" time="1.808">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-OverrideOperationsBinding-ast2" time="0.495">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-20" time="2.873">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORTX12-01" time="3.164">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-04" time="0.357">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-6" time="1.494">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-host-19" time="0.626">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-09" time="0.581">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-11" time="0.474">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-15" time="0.392">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-11" time="1.555">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-02" time="0.435">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTX12-02" time="0.629">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN016" time="1.788">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-06" time="0.347">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-07" time="0.523">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="MsgBus-3" time="1.353">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-02" time="8.485">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-15" time="0.472">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-8" time="1.570">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-24" time="0.465">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBDOUTBTRANS-05" time="2.416">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-OutboundTransportRN" time="2.576">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-10" time="0.354">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GSInfor-2" time="0.555">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOST-01" time="3.127">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-09" time="0.523">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-OperationsNode1" time="0.499">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-BA" time="0.517">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-7" time="1.728">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-11" time="3.265">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-01" time="0.655">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-02" time="0.439">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-07" time="0.344">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCode4" time="0.349">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-11" time="0.544">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN020" time="0.406">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GatewayService-5" time="0.577">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-19" time="0.393">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RN-DeletePARTNER-13" time="0.748">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPE-03" time="0.405">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-BZAgreement-14" time="0.607">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-08" time="0.414">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-1" time="1.819">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-06" time="0.488">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBDBIZID-04" time="1.410">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-04" time="0.435">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="THOST1" time="0.869">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TEnableProtocolHost3" time="2.190">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="MsgBus-4" time="0.380">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-06" time="0.660">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-16" time="0.429">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SCC-05" time="0.404">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RNROLEERROR-01" time="1.243">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-06" time="0.522">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound11" time="0.434">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-11" time="0.347">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN025" time="0.430">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-24" time="0.435">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-14" time="0.351">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSRN-03" time="1.212">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BAErrorCode-03" time="1.350">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-05" time="1.584">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-11" time="0.409">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-4" time="0.391">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-20" time="2.289">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="Role_Group" time="0.406">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-10" time="0.471">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="CONTACT-05" time="0.370">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-07" time="0.447">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXYERROR-09" time="0.333">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-09" time="0.451">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-09" time="0.600">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-04" time="0.713">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN018" time="1.340">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN010" time="0.392">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-02" time="0.375">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-8" time="1.553">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-Update-3" time="1.266">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RBERROR-06" time="0.631">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-10" time="3.097">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-host-15" time="0.796">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound14" time="0.369">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERBA-02" time="1.409">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRS-02" time="1.275">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-08" time="0.517">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN024" time="0.450">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RN-CreatePARTNER-8" time="1.236">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-22" time="0.416">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-05" time="4.811">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Create-operation-2" time="0.872">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN012" time="0.444">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-08" time="1.351">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="INBOUNDTRANSPORT-05" time="0.588">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTBA-02" time="1.840">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-Delete-15" time="0.396">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-06" time="0.571">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPE-01" time="1.031">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GatewayService-4" time="0.442">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-08" time="0.496">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound6" time="2.340">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTX12-01" time="1.827">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSRN-07" time="0.471">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-4" time="1.673">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-07" time="0.338">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN008" time="0.551">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATIONERROR-02" time="1.222">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Create-operation-2" time="0.517">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-EnableProtocol-6" time="2.776">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSX12-03" time="1.456">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGS-20" time="1.609">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-OverrideOperationsBinding-tt1" time="0.635">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-05" time="0.641">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SCC-04" time="0.540">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERLOCATION-01" time="1.613">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOST-20" time="1.871">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSX12-02" time="1.559">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGSX12-04" time="2.275">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN013" time="2.283">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGSX12-01" time="3.109">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-7" time="0.374">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-23" time="0.413">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-9" time="1.760">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-DeleteOperationBinding" time="0.773">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Delete operation node0" time="0.491">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-11" time="0.452">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-07" time="3.525">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-10" time="0.487">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-Create-1" time="1.412">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-BZAgreement-18" time="0.488">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATION-08" time="0.467">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN027" time="0.445">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-22" time="1.329">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-partner-16" time="0.529">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-21" time="1.436">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-OperationsNode0" time="1.447">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-9" time="0.375">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Create-Syncoperation-3" time="2.841">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTBA-03" time="0.401">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PCCEC-08" time="0.367">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-01" time="1.215">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-02" time="0.647">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-09" time="0.517">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Create-operation-1" time="0.887">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-CreateProtocolBind-10" time="2.547">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-03" time="0.390">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-04" time="0.492">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-24" time="0.354">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TBizAgreement21" time="0.525">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXYERROR-08" time="0.377">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-05" time="0.383">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERBA-05" time="0.639">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-02" time="1.005">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-03" time="1.478">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBINDINGSX12-05" time="0.713">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Outboundtrans-10" time="1.688">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLX12-01" time="2.509">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-10" time="0.660">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN005" time="2.099">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-06" time="0.496">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-2" time="0.490">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-9" time="1.610">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-GetAll-7" time="1.059">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINTYPERN-08" time="1.101">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN019" time="0.402">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="LOCATIONERROR-03" time="1.449">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-04" time="0.542">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-07" time="0.500">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTBA-04" time="0.576">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-03" time="0.507">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="HOSTS-21" time="0.849">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-EnableProtocol-7" time="3.266">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOLBA-01" time="3.261">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERBA-01" time="1.687">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Delete-Operation-13" time="0.370">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN023" time="0.400">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-03" time="1.597">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-01" time="3.199">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-Group04" time="0.490">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-06" time="0.540">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Delete Partner" time="0.739">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-19" time="1.519">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-OverrideOperationsBinding-ost2" time="0.541">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ROLE-01" time="9.118">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-01" time="1.925">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBDHOST-04" time="0.721">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-12" time="0.419">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-21" time="0.650">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-15" time="0.622">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="DOMAINPAIRSX12-06" time="1.856">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERX12-01" time="1.339">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Partner" time="1.432">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-09" time="0.428">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-5" time="1.716">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="ENABLEDPROTOCOL-04" time="0.356">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-06" time="2.990">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RosettaNetRoles-Search-6" time="0.310">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="Feature-01" time="11.810">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-07" time="0.431">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-12" time="0.463">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="MsgBus-2" time="0.438">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RBERROR-01" time="0.896">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-2" time="0.373">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="GatewayService-3" time="2.075">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RolesBinding-Delete-14" time="0.559">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN014" time="0.604">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-OperationsRequestResponsetransaction" time="3.019">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="INSTALLED PROTOCOLS-01" time="0.352">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-5" time="0.460">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-12" time="0.531">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="MBERROR-01" time="0.356">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-17" time="0.697">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OBRN-Enableprotocol1" time="3.813">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBD-07" time="2.500">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-14" time="0.381">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-20" time="1.518">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-05" time="3.518">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="USER-15" time="0.513">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-06" time="0.316">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-OverrideOperationsBinding-ast1" time="0.549">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TEnableProtocolPartner4" time="3.194">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="MsgBus-6" time="0.377">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROXY-11" time="1.610">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-01" time="2.389">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RBERROR-02" time="3.489">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATIONS-RN006" time="2.448">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNER-Group03" time="0.630">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-Outboundtrans-8" time="2.216">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="IBTProtocol-3" time="1.541">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PARTNERBA-03" time="0.363">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="UserGroup-03" time="0.463">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OUTBOUNDTRANSPORT-03" time="0.508">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBDBIZID-05" time="0.630">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="TOutbound7" time="0.509">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="BIZAGREEMENT-06" time="0.398">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="SR-22" time="0.456">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="PROTOCOLBD-05" time="3.318">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="RolesBinding-Create-10" time="1.283">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OPERATION-31" time="0.337">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
      <testcase name="OB-EnableProtocol-8" time="2.326">
            <failure message="Unknown" type="Unknown"/>
          </testcase>
  
  
  
</testsuite></testsuites>